Documentation/cmdref/cilium-agent.md

```diff
@@ -25,14 +25,14 @@ cilium-agent [flags]
       --blacklist-conflicting-routes                          Don't blacklist IP allocations conflicting with local non-cilium routes (default true)
       --bpf-compile-debug                                     Enable debugging of the BPF compilation process
       --bpf-ct-global-any-max int                             Maximum number of entries in non-TCP CT table (default 262144)
-      --bpf-ct-global-tcp-max int                             Maximum number of entries in TCP CT table (default 1000000)
+      --bpf-ct-global-tcp-max int                             Maximum number of entries in TCP CT table (default 524288)
```

Comment:
I think there was a reason for this. @joestringer do you remember?

Reply:
Any time we change this setting, we need to be very clear to users as this can cause connection disruption during upgrade if the setting actually changes.

Reply:
Put another way, if users really are saving ~150MB of memory with this patch, then those users are also going to have connection disruption during upgrade.

Reply:
Perhaps it may be better to have a tuning guide in our Cilium documentation where all these knobs could be described?

Reply:
Sounds like it's better to abandon this PR and instead document theses knobs.

Reply:
We need to decide on what is the ideal default and then apply that for new users while we point to the tuning guide for existing users. Right now, even tiny clusters are allocating a huge conntrack table.

Reply:
As discussed in yesterday's Cilium community meeting, I'll add a note to the upgrade guide about this change and document how users can make sure they won't have disrupted connections during upgrade.

I'll also send a separate change which will allow to set the map sizes based on system memory size (probably in a separate PR) and document all the knobs users can use to optimize agent memory usage.